painttiming.bs

```diff
@@ -123,6 +128,7 @@ An [=element=] |target| is <dfn export>contentful</dfn> when one or more of the
 * |target| is a [=canvas=] with its [=context mode=] set to any value other than <code>none</code>.
 * |target| is a [=video element=] that [=represents=] its [=poster frame=] or the first video frame and the frame is available.
 * |target| is an [=svg element with rendered descendants=].
+* |target| is an [=input=] element, with its [=type attribute=] in any state other than [=hidden=].
```

Comment:
How about textarea?

Reply:
It seems a bit fragile to enumerate the list of elements & attributes like this. We also have to enumerate select, progress, meter, etc... I wonder if we can somehow the concept of replaced elements?

Reply:
I think the most precise available word would be [represents](https://html.spec.whatwg.org/multipage/dom.html#represents). From [form elements](https://html.spec.whatwg.org/multipage/form-elements.html#the-datalist-element), "... the datalist element represents nothing ..."

Reply:
OK I found a good definition: [being rendered](https://html.spec.whatwg.org/multipage/rendering.html#being-rendered). Added to "paintable", so:
* all form controls and inputs are "contentful"
* but only ones that are "being rendered" (generate boxes) are "paintable".

Reply:
SGTM

Reply:
Revised. Also, there's no need to define `textarea` here specifically, as `textarea` generates text-nodes, so a non-empty `textarea` would already be considered contentful.